include/deal.II/grid/connectivity.h

```diff
@@ -614,7 +614,7 @@ namespace internal
            {{3, 4, 5, X}},
            {{6, 7, 0, 3}},
            {{7, 8, 1, 4}},
-           {{8, 6, 5, 2}}}};
+           {{8, 6, 2, 5}}}};
```

Comment:
You could probably make a test out of this by using what this function returns to identify the lines that bound each face, then ask where these lines are geometrically located, get their centers, and make sure that these centers are in the order you expect them to?

For every 3d reference cell, if a face is a triangle, I think we assume that the lines are counted clock/counter-clockwise; if a face is a quad, the lines must be in lexicographic order. One could probably check this sort of expectation. Whether it's worth doing that in a separate test is a different matter, but I like these sorts of tests that check qualitative expectations.